package.json

```diff
@@ -30,5 +32,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.1.0"
```

Comment:
This should be already implemented in create-react-app. 
I wanted you to use this syntax. Babel plugins transforms it under the hood. 
Try to uninstall it and see if it works.
Basically its about to use class properties with arrow functions to avoid using bind() each time